Makefile.toml

```diff
@@ -69,6 +69,45 @@ end
 echo "License header check complete"
 '''
 
+[tasks.generate-readmes]
+description = "Automatically generate README.md for each component."
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+glob_pattern = set "./components/**/README.md"
+template = canonicalize README.tpl
+
+# If we specify install_crate we are expected to run a cargo command,
+# not a script, but cargo-readme is not happy with our workspace
+# configuration.
+exec cargo install cargo-readme --version 3.2.0
```

Comment:
Issue: Do we consider it okay for these scripts to be non-hermetic and installing a dependency?  The alternative would be to exit with an error message if the dependency isn't installed yet, like I'm doing here: https://github.com/sffc/omnicu/blob/static-testing/Makefile.toml#L166

Reply:
I would be fine with erroring out, as long as we can insure the dependency is installed for this to run in CI. I'll check.